---
abstract: Pathogen identification is pivotal in diagnosing, treating, and preventing
  diseases, crucial for controlling infections, and safeguarding public health. Traditional
  alignment-based methods, though widely used, are computationally intense and reliant
  on extensive reference databases, often failing to detect novel pathogens due to
  their low sensitivity and specificity. Similarly, conventional machine learning
  techniques, while promising, require large annotated datasets and extensive feature
  engineering and are prone to overfitting. Addressing these challenges, we introduce
  PathoLM, a cutting-edge pathogen language model optimized for the identification
  of pathogenicity in bacterial and viral sequences. Leveraging the strengths of pre-trained
  DNA models such as the Nucleotide Transformer, PathoLM requires minimal data for
  fine-tuning, thereby enhancing pathogen detection capabilities. It effectively captures
  a broader genomic context, significantly improving the identification of novel and
  divergent pathogens. We developed a comprehensive data set comprising approximately
  30 species of viruses and bacteria, including ESKAPEE pathogens, and seven notably
  virulent bacterial strains resistant to antibiotics. Additionally, we curated a
  species classification dataset centered specifically on the ESKAPEE group. In comparative
  assessments, PathoLM dramatically outperforms existing models like DciPatho, demonstrating
  robust zero-shot and few-shot capabilities. Furthermore, we expanded PathoLM-Sp
  for ESKAPEE species classification, where it showed superior performance compared
  to other advanced deep learning methods, despite the complexities of the task.
title: 'PathoLM: Identifying pathogenicity from the DNA sequence through the Genome
  Foundation Model'
layout: inproceedings
series: Proceedings of Machine Learning Research
publisher: PMLR
issn: 2640-3498
id: dip24a
month: 0
tex_title: 'PathoLM: Identifying pathogenicity from the DNA sequence through the Genome
  Foundation Model'
firstpage: 153
lastpage: 161
page: 153-161
order: 153
cycles: false
bibtex_author: Dip, Sajib Acharjee
author:
- given: Sajib Acharjee
  family: Dip
date: 2024-11-17
address:
container-title: Proceedings of the 19th Machine Learning in Computational Biology
  meeting
volume: '261'
genre: inproceedings
issued:
  date-parts:
  - 2024
  - 11
  - 17
pdf: https://raw.githubusercontent.com/mlresearch/v261/main/assets/dip24a/dip24a.pdf
extras: []
# Format based on Martin Fenner's citeproc: https://blog.front-matter.io/posts/citeproc-yaml-for-bibliographies/
---
